h2o-algos/src/main/java/hex/tree/DHistogram.java

```diff
@@ -417,7 +417,7 @@ public void updateHisto(double[] ws, double[] cs, double[] ys, int [] rows, int
       double y = ys[k];
       assert (!Double.isNaN(y));
       double wy = weight * y;
-      double wyy = wy * y;
+      double wyy = wy * y;  // need to verify this is correct or if not, fix it.
```

Comment:
please add a FIXME with reference to the jira (this will make sure the comment is not lost & forgotten)